### DISCUSSION TITLE
How do you add Slot Number to custom /sethome command?

### DISCUSSION THREAD

**Discussion by redalpha123123** [Zrips/CMI#5132] [Q&A]
I have a custom command for /cmi set sethome. It is /sethome. Simple right? The problem is that when I try to add a Slot number after /sethome  (IE: /sethome House 2) it sets it as Home for slot 1.

How do I set if up to make slot mumbers?

**Comment by CloudeLecaw** [comment]
the command written in wiki is:

> cmi sethome (homeName) (playerName) (-p) (-l:worldName;x;y;z) (block/Material) (slotNumber)

your alias should be something like
```
  sethomealias:
    Cmds:
    - perm:cmi.command.customalias.sethomealias@! msg! [playerName] &cYou cant use this command.
    - perm:cmi.command.customalias.sethomealias#! check:$2==null! asConsole! cmi sethome $1 [playerName]
	- check:$2!=null! asConsole! cmi sethome $1 [playerName] $2
```
the permission your player need to use it would be `cmi.command.customalias.sethomealias`
The alias checks if a slotnumber is defined. if no slotnumer is defined it adds the home for the player
if a slotnumber is defined then it sets the home at the defined slot
